packages/passage/src/stimulus-tabs.jsx

```diff
@@ -33,37 +31,16 @@ const styles = (theme) => ({
   buttonContainer: {
     background: color.background(),
   },
+  tabContainer: {
+    backgroundColor: color.background(),
+    color: color.text(),
+    padding: theme.spacing.unit * 2,
```

Comment:
have you had in mind that padding was different based on props.multiple?

Reply:
I changed the padding in order to align with the last UI changes on authoring side